ocs_ci/utility/gcp.py

```
@@ -75,6 +75,210 @@ def load_service_account_key_dict(filepath=SERVICE_ACCOUNT_KEY_FILEPATH):
+
+    Returns:
+        str: The created service account's email address
+
+    """
+    os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = SERVICE_ACCOUNT_KEY_FILEPATH
+    iam = build("iam", "v1", cache_discovery=False)
+    crm = build("cloudresourcemanager", "v1", cache_discovery=False)
+    project_resource = f"projects/{gcp_project_id}"
+
+    # 1. Create the GCP service account
+    logger.info(f"Creating GCP service account {sa_name} in project {gcp_project_id}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)